includes/class-wc-tax.php

```diff
@@ -679,6 +679,16 @@ public static function get_rate_label( $key_or_rate ) {
 	 * @return  string
 	 */
 	public static function get_rate_percent( $key_or_rate ) {
+		return apply_filters( 'woocommerce_rate_percent', self::get_rate_percent_value( $key_or_rate ) . '%', is_object( $key_or_rate ) ? $key_or_rate->tax_rate_id : $key_or_rate );
```

Comment:
just a suggestion, maybe refactor this line to figure out rate id in separate lines above so that this is more readable.